homework/homework150124.py

```diff
@@ -0,0 +1,37 @@
+"""
+HOMEWORK8.
+
+Task Description:
+-----------------
+HOMEWORK8 involves the following task.
+Given 0 < number(int) <= 11
+Need to print multiplication table based on this number.
+Output example for num 5:
+1   2   3   4   5
+2   4   6   8  10
+3   6   9  12  15
+4   8  12  16  20
+5  10  15  20  25
+"""
+
+
+def processing_table(num):
+    """
+    Print a multiplication table for the given number.
+
+    Parameters:
+    - num (int): The number for which to generate the multiplication table.
+    Should be between 1 and 11 (inclusive).
+    """
+    if 0 < num <= 11:
```

Comment:
Such checks redundant a little bit  - better just set num=11 and then code 
Without any input etc. In real automation scripts parameter we send not by interactive typing but via script arguments 
But this is theory from future lessons